src/Lumi/Components/Upload.purs

```diff
@@ -124,6 +126,8 @@ type UploadProps =
 
 type UploadBackend =
   { fetch :: FileId -> Aff FileInfo
+    -- | Allows for running arbitrary 
```

Comment:
Is there suppose to be more to this comment? As is it doesn't seem that useful.

Reply:
Oh yes, thanks for catching that!